code/game/area/area.dm

```diff
@@ -326,3 +330,56 @@
 
 /area/return_gas()
 	return gas_type
+
+///Set this area as a contested zone, that will monitors which faction controls it.
+/area/proc/set_to_contested()
+	RegisterSignal(src, COMSIG_AREA_ENTERED, .proc/add_faction_member)
+	RegisterSignal(src, COMSIG_AREA_EXITED, .proc/left_area)
+	minimap_color = MINIMAP_AREA_CONTESTED_ZONE
```

Comment:
Is therea  particular reason this doesnt crash if theminimaps are already generated

Reply:
why would this crash, at worst this doens't do anything.

And minimap is one of the last SS to init

Reply:
I can spawn these after ssminimaps inits

Reply:
ok done